src/layouts/default/PlayerOSD/PlayerFullscreen.vue

```diff
@@ -489,6 +502,13 @@ const radioNameClick = function (item: Radio | ItemMapping) {
   store.showFullscreenPlayer = false;
 };
 
+const radioTitleClick = function (streamTitle: string) {
+  // radio station title clicked
+  store.globalSearchTerm = streamTitle;
+  router.push({ name: 'search' });
```

Comment:
doesn't this issue a warning if the route is already active ? Keep an eye on it, if you see an error reported by the vue router then this needs to be guarded.